spec/workers/atmosphere/vm_template_monitoring_worker_spec.rb

```diff
@@ -13,8 +13,12 @@
     it { should be_processed_in :monitoring }
   end
 
-  context 'updating cloud site images' do
-    let(:cyfronet_folsom) { create(:compute_site, site_id: 'cyfronet-folsom',config: '{"provider": "openstack", "openstack_auth_url": "http://10.100.0.2:5000/v2.0/tokens", "openstack_api_key": "key", "openstack_username": "user"}') }
+  context 'updating tenant images' do
+    let(:cyfronet_folsom) do
+      create(:tenant, tenant_id: 'cyfronet-folsom', \
+      config: '{"provider": "openstack", "openstack_auth_url": "http://10.100.0.2:5000/v2.0/tokens",
```

Comment:
Line is too long. [100/80]<br>Align the elements of a hash literal if they span more than one line.